panel/pane/media.py

```diff
@@ -143,8 +165,11 @@ class Audio(_MediaBase):
 
     @classmethod
     def applies(cls, obj: Any) -> float | bool | None:
+        if isinstance(obj, TorchLike):
+            obj = obj.numpy()
```

Comment:
This scares me as it can potentially make in-memory copies of huge array, just to check if their dtype matches.

Reply:
😁i Think i can ind a better implementation. We should just check if it Torchlike and then the dimension

Reply:
I believe your concern has now been adressed.